### PR TITLE
Introspect table comments in Doctrine\DBAL\Schema\Table when generating schema

### PR DESCRIPTION
**Pull request by mondrake**
This is an attempt at implementing https://github.com/doctrine/dbal/issues/1335.
First attempt at a patch here, so please bear with me :)

### CODE REVIEW

File: lib/Doctrine/DBAL/Schema/Table.php [hunk 1]
```diff
@@ -794,6 +794,16 @@ public function getOptions()
     }
 
     /**
+     * Gets the table comment if available.
+     *
+     * @return string|null
+     */
+    public function getComment()
+    {
+        return $this->hasOption('comment') ? $this->getOption('comment') : NULL;
```
Comment (phansys) [inline lib/Doctrine/DBAL/Schema/Table.php]: Please, use `null` instead.

File: tests/Doctrine/Tests/DBAL/Functional/Schema/MySqlSchemaManagerTest.php [hunk 2]
```diff
@@ -319,4 +319,16 @@ public function testListFloatTypeColumns()
...
+
+    public function testIntrospectTableComment()
+    {
+        $table = new Table("introspect_table_comment");
+        $table->addColumn('foo_id', 'integer');
+        $table->addColumn('bar_id', 'integer');
+        $table->addOption('comment', 'This is a comment at table level.');
+        $this->_sm->createTable($table);
+
+        $tableFetched = $this->_sm->listTableDetails("introspect_table_comment");
```
Comment (phansys) [inline tests/Doctrine/Tests/DBAL/Functional/Schema/MySqlSchemaManagerTest.php]: Please, use single quotes.

File: tests/Doctrine/Tests/DBAL/Functional/Schema/MySqlSchemaManagerTest.php [hunk 3]
```diff
@@ -319,4 +319,16 @@ public function testListFloatTypeColumns()
...
+        $this->assertEquals('This is a comment at table level.', $tableFetched->getComment());
```
Comment (phansys) [inline tests/Doctrine/Tests/DBAL/Functional/Schema/MySqlSchemaManagerTest.php]: Please, use `assertSame()` instead.

**Review by phansys — CHANGES_REQUESTED** [review]
Status: Needs work.

**Comment by mondrake** [comment]
Thanks for review @phansys , I made the changes requested.

**Pull request closed without merging**